Fastfile

```diff
@@ -75,115 +73,113 @@ end
+  ENV['SCAN_DEVICE'] = 'iPhone 6 (9.3)' if ENV['SCAN_DEVICE'].nil?
+  xcode_select(ENV['TAB_XCODE_PATH']) if is_ci && !ENV['TAB_XCODE_PATH'].nil?
+
+  unless ENV['TAB_UI_TEST_SCHEME'].nil?
+    ENV['TAB_REPORT_FORMATS'] = 'html' if ENV['TAB_OUTPUT_TYPES'].nil?
+    ENV['TAB_UI_TEST_DEVICES'] = 'iPhone 8' if ENV['TAB_UI_TEST_DEVICES'].nil?
   end
 end
 
-def _buildAndDeployToHockey()
+def _build_and_deploy_to_hockey
   icon_overlay(version: get_version_number)
-  _set_build_number()
-  _build_ipa()
-  _upload_to_hockey()
+  _set_build_number
+  _build_ipa
+  _upload_to_hockey
 end
 
-def _set_build_number()
-  build_number = "0"
+def _set_build_number
   use_timestamp = ENV['TAB_USE_TIME_FOR_BUILD_NUMBER'] || false
-  if use_timestamp
-    build_number = Time.now.strftime("%y%m%d%H%M")
-  else
-    build_number = ENV['BUILD_NUMBER']
-  end
+  build_number =
+    if use_timestamp
+      Time.now.strftime("%y%m%d%H%M")
+    else
+      ENV['BUILD_NUMBER']
+    end
   increment_build_number(build_number: build_number)
 end
 
-def _build_ipa()
+def _build_ipa
   update_app_identifier(xcodeproj: ENV['FL_UPDATE_PLIST_PROJECT_PATH'],
-                       plist_path: ENV['FL_UPDATE_PLIST_PATH'],
-                   app_identifier: ENV['FL_UPDATE_PLIST_APP_IDENTIFIER'] )
+                        plist_path: ENV['FL_UPDATE_PLIST_PATH'],
+                        app_identifier: ENV['FL_UPDATE_PLIST_APP_IDENTIFIER'])
   update_info_plist
-  _build_with_gym()
+  _build_with_gym
 end
 
-def _build_with_gym()
+def _build_with_gym
   install_provisioning_profile
   _update_team_id_if_necessary
-  export_method = _get_export_method()
-  xcconfig_filename = Dir.pwd + "/TAB.release.xcconfig"
+  export_method = _get_export_method
+  xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
   create_xcconfig(filename: xcconfig_filename)
   gym(export_method: export_method, xcconfig: xcconfig_filename)
 end
 
-def _get_export_method()
-  ENV['GYM_EXPORT_OPTIONS'].nil? ? _fallback_to_enterprise_export_method() : get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "method")
+def _get_export_method
+  if ENV['GYM_EXPORT_OPTIONS'].nil?
+    _fallback_to_enterprise_export_method
+  else
+    get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: 'method')
+  end
 end
 
-def _fallback_to_enterprise_export_method()
-  UI.message("Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined")
-  "enterprise"
+def _fallback_to_enterprise_export_method
+  UI.message('Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined')
+  'enterprise'
 end
 
-def _update_team_id_if_necessary()
+def _update_team_id_if_necessary
   project_path = ENV['FL_PROJECT_SIGNING_PROJECT_PATH']
-  team_id = _get_team_id()
+  team_id = _get_team_id
   if !project_path.to_s.strip.empty? && !team_id.to_s.strip.empty?
     UI.message("Updating project team with project path '#{project_path}' and team id '#{team_id}'.")
```

Comment:
" instead of '

Reply:
I'm unsure about this one - as I'm string interpolating  with #{project_path} but I'm not 100% sure

Reply:
```UI.message('Updating project team with project path %{project_path} and team id %{team_id}.')```

I'm pretty sure this is valid and will work

Reply:
Still need the `'` around `#{project_path}` though so it reads nicely in the logs. Pedantic I know.